cdi/src/main/java/org/hibernate/validator/internal/cdi/ValidatorFactoryBean.java

```diff
@@ -51,7 +53,7 @@
 
 	public ValidatorFactoryBean(BeanManager beanManager, ValidationProviderHelper validationProviderHelper) {
 		this.beanManager = beanManager;
-		this.destructibleResources = newHashSet( 4 );
+		this.destructibleResources = newHashSet( 5 );
```

Comment:
Yeah, the issue is that we often do CollectionHelper.newCollection( otherCollection.size() ). And I won't expect people to do the math by themselves.

I'll cherry-pick the OGM commit to fix this once and for all.